tools/jenkins/run_performance.sh

```diff
@@ -34,4 +34,4 @@ set -ex
 # Enter the gRPC repo root
 cd $(dirname $0)/../..
 
-tools/run_tests/run_performance_tests.py -l c++ node ruby csharp python
+tools/run_tests/run_performance_tests.py -l c++ node ruby csharp python --netperf --category smoketest
```

Comment:
Actually, I cannot run go as part of smoketest yet (because that would require setting up a multi-repo project). Removing go for now.